### PR TITLE
Avoid dimming when moving mouse / pen around the screen

### PR DESCRIPTION
**Pull request by nirvn**
Tested QField using bluetooth mouse and keyboard revealed this shortcoming.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for fa769d448cdbb8db92aac01202177181ea52b6ad:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2568-27911d7fe4dfa5d97900a45107951db572d3b2ae-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2568-27911d7fe4dfa5d97900a45107951db572d3b2ae-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2568-27911d7fe4dfa5d97900a45107951db572d3b2ae-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2568-27911d7fe4dfa5d97900a45107951db572d3b2ae-x86-android.apk)

File: src/core/screendimmer.cpp [hunk 1]
```diff
@@ -66,7 +66,7 @@ void ScreenDimmer::setSuspend( bool suspend )
 
 bool ScreenDimmer::eventFilter( QObject *obj, QEvent *event )
 {
-  if ( event->type() == QEvent::KeyPress || event->type() == QEvent::MouseButtonPress || event->type() == QEvent::TouchBegin || event->type() == QEvent::InputMethod )
+  if ( event->type() == QEvent::KeyPress || event->type() == QEvent::MouseMove || event->type() == QEvent::MouseButtonPress || event->type() == QEvent::TabletMove || event->type() == QEvent::TabletPress || event->type() == QEvent::TouchBegin || event->type() == QEvent::TouchUpdate || event->type() == QEvent::InputMethod || event->type() == QEvent::Wheel )
```
[the remark this answers is not in the document]
Comment (suricactus) [reply src/core/screendimmer.cpp:69]: Also, caching `event->type()` in a var might save a few CPU cycles :)

[the remark this answers is not in the document]
Comment (nirvn) [reply src/core/screendimmer.cpp:69]: @suricactus , I had it on multiple lines but the auto formatting glued it back into one line.

[the remark this answers is not in the document]
Comment (suricactus) [reply src/core/screendimmer.cpp:69]: crap, there should be a line length cap...

**Review by suricactus — APPROVED** [review]

**Comment by nirvn** [comment]
@suricactus , applied your CPU cycle saving suggestion.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for f1005ecb1858760d1b3494be40ae23848927e5e1:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2568-01159e798f39f2935eb1efa5c784f622f8d95050-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2568-01159e798f39f2935eb1efa5c784f622f8d95050-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2568-01159e798f39f2935eb1efa5c784f622f8d95050-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2568-01159e798f39f2935eb1efa5c784f622f8d95050-x86-android.apk)

**Pull request merged**